services/site/src/components/common/dropdown/Dropdown.tsx

```diff
@@ -72,9 +68,9 @@ type DropdownComponent = React.FC<React.PropsWithChildren<DropdownProps>> & {
   Group: GroupComponent;
 };
 
-const Dropdown: DropdownComponent = ({ children, triggerButton }) => {
+const Dropdown: DropdownComponent = ({ children, triggerButton, hamburger, isSticky }) => {
```

Comment:
I'd much prefer `isMobile` over `hamburger`. The way I see it, the hamburger icon is just a symptom, not the condition